README.md

```diff
@@ -167,6 +174,36 @@ PUPPERWARE_ANALYTICS_ENABLED=false
 
 This file is in the `.gitignore` file and will not be managed or changed by pupperware.
 
+## Environment
+
+Environment variables may be specified in `.env` file in this
+directory.
+
+### DNS_ALT_NAMES
+
+See *Provisioning* above.
+
+### DOMAIN
+
+See *Provisioning* above.
+
+### PUPPETWARE_ANALYTICS_ENABLED
+
+See *Analytics Data Collection* below
+
+### PUPPETSERVER_PORT
```

Comment:
I don't have a great idea for a way to make this variable name more clear, but with the name `PUPPETSERVER_PORT` my initial thought was that this was going to change the port the container was listening to, not the mapped port on the host running the container. Would like to find a name that makes it a little more clear that this is the mapped port.